doc/langref.html.in

```diff
@@ -7952,6 +7952,25 @@ pub const TypeInfo = union(TypeId) {
       {#link|pointer|Pointers#}.
       </p>
       {#header_close#}
+
+      {#header_open|@shuffle#}
+      <pre>{#syntax#}@shuffle(comptime ElemType: type, a: @Vector(mask.len, ElemType), b: @Vector(mask.len, ElemType), comptime mask: @Vector(_, u32)) @Vector(mask.len, ElemType){#endsyntax#}</pre>
+      <p>
+      Does the {#syntax#}shufflevector{#endsyntax#} instruction. Each element in {#syntax#}comptime{#endsyntax#}
```

Comment:
The `shufflevector` instruction is not standard or widely known, so it should be defined here.

Reply:
It is defined in the next sentence....